### ISSUE TITLE
Document running druid without zk

### ISSUE THREAD

**Issue by AdheipSingh** [datainfrahq/druid-operator#15]

**Comment by danielloader** [comment]
Would be helpful, thanks for raising a ticket!

**Comment by AdheipSingh** [comment]
@danielloader yes its helpful. Currently i am adding this extension in sync with druid operator on apache druid integration tests. So it get validated against k8s API version. Once done, will doc, as it will be validated on apache druid repo.
